multimedia/VLC2/Portfile

```diff
@@ -175,6 +175,10 @@ if {(${subport} eq ${name}) || (${subport} eq "lib${name}")} {
     } elseif {[string match *gcc* ${configure.compiler}]} {
         patchfiles-append \
                         0001-Fixes-for-modern-GCC.patch
+        # codec/avcodec/encoder.c:883:41: error: passing argument 2 of 'vlc_av_get_options_ng'
+        # from incompatible pointer type [-Wincompatible-pointer-types]
+        configure.cflags-append \
+                        -Wno-incompatible-pointer-types
```

Comment:
I really hate removing compiler warnings. I think it's important to know why something fails and not just decide to ignore the compiler.

Reply:
This is an old version of VLC, it is not supported by upstream.

Reply:
Fine, then patch the code instead of ignoring the warning?

Reply:
@pmetzger If you suggest a fix, I will happily add it.

Reply:
Read the code at that place. Is the pointer being passed safe to pass? If so, add a coercion. Is it unsafe to pass? If so, figure out what sort of patch would render the code correct.

Reply:
@pmetzger Thank you for insisting here, I think we actually had an error in one patch. I will try sorting that out now.

Reply:
@pmetzger It took way longer than expected, but now VLC2 builds with gcc14 without converting errors into warnings.